exercises/rxjs-observable/README.md

```diff
@@ -0,0 +1,34 @@
+# RxJS Exercises
+
+## Exercise 1
+
+Create an RxJS `Observable` to display 2 different integers using `setInterval` and `next`.
+
+## Exercise 2
+
+Using the `map` operator create an RxJS `Observable` that will take an array of numbers and multiply each item by 2. 
```

Comment:
To more clearly indicate what needs to be done, maybe reword to something similar: Create an observable from an array of numbers and using `map`, multiply each item by 2